### DISCUSSION TITLE
External network request to qq.com (203.205.255.185)

### DISCUSSION THREAD

**Discussion by Tundra-bit** [TT-ReBORN/Georgia-ReBORN#149] [Q&A] [answered]
Howdy,

 First off, thanks for making Foobar great again, really enjoying Georgia Reborn!

   I've installed the latest release Georgia-ReBORN_v3.0-RC2.  I then realized my firewall was complaining about access to **y.qq.com** (203.205.255.185), according to VirusTotal it is non malicious. 

My question is, besides renaming qqmusic.js, is there a GUI method to disable these type service requests?

And, is there a location where I can get more information about them?

Thanks!

**Comment by TT-ReBORN** [comment, marked as answer]
Hi @Tundra-bit,

it's good to hear you're enjoying Georgia-ReBORN. 

Here is information about QQ: https://en.wikipedia.org/wiki/Tencent_QQ
It doesn't matter which social media platform you name or other big ( even small ) websites, 
they all want your information as much as possible ( popups, cookies ), it is the new gold after all...

But I wouldn't worry too about QQ because you are only fetching lyrics from the server and not using any of their other services.

You can always disable lyric sources via:
`foobar2000` > `Preferences` > `Tools` > `ESLyric` > `Lyric Option` > `Lyric Sources`

-TT

**Comment by Tundra-bit** [comment]
Thank you!
